lib/icinga/downtime.cpp

```diff
@@ -208,7 +206,7 @@ int Downtime::GetNextDowntimeID()
 	return l_NextDowntimeID;
 }
 
-String Downtime::AddDowntime(const Checkable::Ptr& checkable, const String& author,
+Downtime::Ptr Downtime::AddDowntime(const Checkable::Ptr &checkable, const String& author,
```

Comment:
`s/T& x/T &x/` is TBD and another construction area.

Reply:
Didn't want to start that discussion, I blame IDE auto-formatting :D

Reply:
Which IDE auto-formatting?

Reply:
Haven't bothered with configuring Icinga coding style in detail, so it still tends to attach `&` to the name instead of the type, this probably happened here as I undid some style changes and probably forgot this one.